src/messages/categoryChanged.js

```diff
@@ -0,0 +1,16 @@
+function notifyCategoryChanged(client, streamer, streamChannelName, streamStatus) {
+    console.log(`notification de live pour ${streamer} : category changed`);
+    const channel = client.channels.cache.find(
+      (ch) => ch.name === streamChannelName
+    );
+    if (channel) {
+      channel.send(
+        `🚀 @everyone ${streamer.replace('_', '\\_')} change de jeu. il est maintenant sur ${streamStatus[`${streamer}_category`]} ! Regardez-le ici: https://www.twitch.tv/${streamer}`
```

Comment:
For me, name problem is also on the link

Reply:
Maybe we should use String.prototype.replaceAll or String.prototype.replace(/_/g, "\\_") because without global flag, it'll only replace the first "_"